src/git/_tags

```diff
@@ -0,0 +1,4 @@
+true: safe_string, bin_annot, debug, keep_locs, use_menhir
+<*/**>: warn(@20@25@26@33@39)
+
+<*/**.{ml,mli,byte,native}>: package(fpath cstruct lwt lwt.unix angstrom faraday farfadet digestif.c bos decompress lru camlzip uri astring), linkdep(rabin_stubs.o)
```

Comment:
Yes, it's to compile `main.ml`, I will delete this - for sure, the library don't have these dependence (`bos` was no longer necessary, `lwt.unix` is for `lwt_fs_unix.ml` and `digestif.c` is for `sha1.ml`.